wpimath/src/main/java/edu/wpi/first/math/filter/LinearFilter.java

```diff
@@ -264,4 +266,13 @@ private static int factorial(int n) {
       return n * factorial(n - 1);
     }
   }
+
+  @Override
+  public void initSendable(SendableBuilder builder) {
+    builder
+        .addDoubleArrayProperty("inputGains", () -> m_inputGains, null)
+        .addDoubleArrayProperty("outputGains", () -> m_outputGains, null)
+        .addDoubleArrayProperty("inputs", m_inputs::getData, null)
+        .addDoubleArrayProperty("outputs", m_outputs::getData, null);
```

Comment:
Since this is a circular buffer, the beginning of the input and output buffers in LinearFilter aren't the same as the beginning of the underlying storage. Use a for loop up to the size and get() instead.

Reply:
Also, per discussion on discord, we should remove the gains and just publish the latest output (not all of them), since the user mainly cares about plotting the filter output over time.